src/SDKs/SqlManagement/Management.Sql/Customizations/DatabaseVulnerabilityAssessmentRuleBaselinesOperationsExtensions.cs

```diff
@@ -0,0 +1,196 @@
+    {
+            /// <summary>
+            /// Gets a database's vulnerability assessment rule baseline.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='resourceGroupName'>
+            /// The name of the resource group that contains the resource. You can obtain
+            /// this value from the Azure Resource Manager API or the portal.
+            /// </param>
+            /// <param name='serverName'>
+            /// The name of the server.
+            /// </param>
+            /// <param name='databaseName'>
+            /// The name of the database for which the vulnerability assessment rule
+            /// baseline is defined.
+            /// </param>
+            /// <param name='ruleId'>
+            /// The vulnerability assessment rule ID.
+            /// </param>
+            public static DatabaseVulnerabilityAssessmentRuleBaseline Get(this IDatabaseVulnerabilityAssessmentRuleBaselinesOperations operations, string resourceGroupName, string serverName, string databaseName, string ruleId)
+            {
+                return operations.GetAsync(resourceGroupName, serverName, databaseName, ruleId).GetAwaiter().GetResult();
+            }
+
+            /// <summary>
+            /// Gets a database's vulnerability assessment rule baseline.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='resourceGroupName'>
+            /// The name of the resource group that contains the resource. You can obtain
+            /// this value from the Azure Resource Manager API or the portal.
+            /// </param>
+            /// <param name='serverName'>
+            /// The name of the server.
+            /// </param>
+            /// <param name='databaseName'>
+            /// The name of the database for which the vulnerability assessment rule
+            /// baseline is defined.
+            /// </param>
+            /// <param name='ruleId'>
+            /// The vulnerability assessment rule ID.
+            /// </param>
+            /// <param name='cancellationToken'>
+            /// The cancellation token.
+            /// </param>
+            public static async Task<DatabaseVulnerabilityAssessmentRuleBaseline> GetAsync(this IDatabaseVulnerabilityAssessmentRuleBaselinesOperations operations, string resourceGroupName, string serverName, string databaseName, string ruleId, CancellationToken cancellationToken = default(CancellationToken))
+            {
+                using (var _result = await operations.GetWithHttpMessagesAsync(resourceGroupName, serverName, databaseName, ruleId, VulnerabilityAssessmentPolicyBaselineName.Default, null, cancellationToken).ConfigureAwait(false))
```

Comment:
This is still more complicated than it needs to be :) Just call the auto-generated GetAsync(resourceGroupName, serverName, databaseName, ruleId, VulnerabilityAssessmentPolicyBaselineName.Default)

Reply:
Done